src/com/massivecraft/vampire/ListenerMain.java

```diff
@@ -603,6 +604,61 @@ public void foodBlood(EntityDamageEvent event)
 	}
 	
 	// -------------------------------------------- //
+	// BLOOD FLASK
+	// -------------------------------------------- //
+	
+	@EventHandler(priority = EventPriority.NORMAL)
+	public void bloodFlaskConsume(PlayerItemConsumeEvent event)
+	{
+		// If the item is a potion ...
+		ItemStack item = event.getItem();
+		if ( ! item.getType().equals(Material.POTION)) return;
+		
+		// ... and is a blood flask ...
+		if ( ! BloodFlaskUtil.isBloodFlask(item)) return;
+		
+		// ... get the blood amount ...
+		double amount = BloodFlaskUtil.getBloodFlaskAmount(item);
+		
+		// ... and is the blood vampiric?  ...
+		boolean isVampiric = BloodFlaskUtil.getBloodFlaskVampiric(item);
+		
+		// ... get the player ...
+		UPlayer vme = UPlayer.get(event.getPlayer());
+		
+		// ... if the player is a vampire, are they bloodlusting? ...
+		if (vme.isBloodlusting())
+		{
+			vme.msg(MLang.get().donateBloodlusting);
+			event.setCancelled(true);
+			return;
+		}
+		
+		// ... calculate and add the blood amount to the player ...
+		double lacking;
+		if (vme.isVampire())
+		{
+			// Vampires drink blood to replenish food.
+			lacking = (20-vme.getFood().get());
+			if (amount>lacking) { amount = lacking; }

```

Comment:
You don't need to put the brackets around the statement. The compiler actually understands it without.